build.sh

```diff
@@ -15,6 +15,22 @@ unset CALLBACK
 
 # Run build process based on configuration files
 
+# load nvm
+. "$NVM_DIR/nvm.sh"
+
+# use .nvmrc if it exists
+if [[ -f .nvmrc ]]; then
+  nvm install
```

Comment:
This seems to cause the same issues we've seen with the `PREFIX` env variable:

```
# unset PREFIX
# nvm use
```

We could unset and reset it here like we do in the Dockerfile, or we could hold off on merging this until we rename `PREFIX` to `SITE_PREFIX`.